src/pages/edit_resource/description_step/TagsField.tsx

```diff
@@ -17,6 +19,23 @@ interface ResourceWrapper {
 interface TagsFieldProps {
   setAllChangesSaved: (status: boolean) => void;
 }
+const StyledChip = styled(Chip)`
+  && {
```

Comment:
denne var ny. er det sass ?

Reply:
ja, er sass.
https://css-tricks.com/the-sass-ampersand/